aiogram_dialog/manager/message_manager.py

```diff
@@ -91,6 +91,34 @@ async def show_message(
             return await self.send_message(bot, new_message)
 
         return await self.edit_message_safe(bot, new_message, old_message)
+    
+    # New text (Shao-mod)
+    async def new_text(self, bot: Bot, new_text: str, old_message: Optional[Message]):
+        if not old_message:
+            return
+        logger.debug("new_text in %s", old_message.chat)
+        try:
+            if old_message.photo:
```

Comment:
And if `old_message.audio` is present? Probably check `if message.caption` here?

But media can be without captions. What if check here content-type of message?